cmd/osquery-perf/agent.go

```diff
@@ -755,12 +739,12 @@ func (a *agent) processQuery(name, query string) (handled bool, results []map[st
 			results = a.batteries()
 		}
 		return true, results, &ss
-	case name == hostDetailQueryPrefix+"os_unix_like":
-		ss := fleet.OsqueryStatus(rand.Intn(2))
-		if ss == fleet.StatusOK {
-			results = a.osUnixLike()
-		}
-		return true, results, &ss
+	// case name == hostDetailQueryPrefix+"os_unix_like":
+	// 	ss := fleet.OsqueryStatus(rand.Intn(2))
+	// 	if ss == fleet.StatusOK {
+	// 		results = a.osUnixLike()
+	// 	}
+	// 	return true, results, &ss
```

Comment:
Why?

Reply:
This was moved into the template and I forgot to delete the commented code.